crates/cxx-qt-gen/test_inputs/properties.rs

```diff
@@ -12,7 +12,13 @@ mod ffi {
         #[qproperty(i32, primitive)]
         #[qproperty(QPoint, trivial)]
         #[qproperty(i32, custom_function_prop, READ = my_getter, WRITE = my_setter, NOTIFY)]
-        #[qproperty(i32, readonly_prop, READ)]
+        #[qproperty(
+            i32,
+            readonly_prop,
+            READ,
+            cxx_name = "renamedProperty",
+            rust_name = "renamed_property"
```

Comment:
maybe call these renamed_readonly_property and renamedReadonlyProperty ?

Reply:
Done, made them write too so we can see the name of the setter

Reply:
Now it's not read-only though :-D maybe instead of editing the existing readonly_prop, revert that to how it was and add a new property with a custom read/write/notify called renamed_property / renamedProperty to check all cases?

Reply:
Fair point, don't know why I specifically picked that one to begin with